### PR TITLE
Update 2017-04-09-sutherland-benjelene.md

### LINKED ISSUE #245: Update Benjelene Sutherland's profile
Benji's now a grad student. Plus she will have a bunch of new profile info/links once she completes the onboarding stuff.

### PR DESCRIPTION
**Pull request by Benjelene**
Hi @tsalo ! here is the new PR. Thank you! Closes #245

### CODE REVIEW

File: team/_posts/2017-04-09-sutherland-benjelene.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ alumni: false
 cv:
 email: bsuth005@fiu.edu
 github:
-orcid:
+orcid:https://orcid.org/0000-0003-1721-9554
```
Comment (tsalo) [inline team/_posts/2017-04-09-sutherland-benjelene.md]:
```suggestion
orcid: 0000-0003-1721-9554
```
You don't need the full URL, plus there should be a space after the field and before the value.

File: team/_posts/2017-04-09-sutherland-benjelene.md [hunk 2]
```diff
@@ -13,12 +13,12 @@ alumni: false
...
 osf:
 publons:
-researchgate:
+researchgate:https://www.researchgate.net/profile/Benjelene_Sutherland
```
Comment (tsalo) [inline team/_posts/2017-04-09-sutherland-benjelene.md]:
```suggestion
researchgate: Benjelene_Sutherland
```
You just need the username (i.e., the last part of the URL).

**Review by tsalo — CHANGES_REQUESTED** [review]
There are minor problems with two of the fields you've updated. I have added code suggestions that you can either accept or emulate in a separate commit.

You can also automatically close specific issues with pull requests. There is an open issue about your profile (#245). Please add the phrase "Closes #245" to the first comment in the PR above to link the two.

File: team/_posts/2017-04-09-sutherland-benjelene.md [hunk 3]
```diff
@@ -13,10 +13,10 @@ alumni: false
 cv:
 email: bsuth005@fiu.edu
 github:
-orcid: 0000-0003-1721-9554
+orcid:0000-0003-1721-9554
```
Comment (tsalo) [inline team/_posts/2017-04-09-sutherland-benjelene.md]: We actually want this space.

File: team/_posts/2017-04-09-sutherland-benjelene.md [hunk 4]
```diff
@@ -13,10 +13,10 @@ alumni: false
...
 osf:
 publons:
-researchgate: Benjelene_Sutherland
+researchgate:Benjelene_Sutherland
```
Comment (tsalo) [inline team/_posts/2017-04-09-sutherland-benjelene.md]: We want this space too.

**Review by Benjelene — COMMENTED** [review]
Good morning @tsalo !

I hope all is well. Is this okay? Thank you!

**Comment by tsalo** [comment]
The profile looks good. The only thing left to do is update your original comment (not the title, but the first box you see on this page) with the term "Closes #245".

**Pull request merged**